index.js

```diff
@@ -43,22 +52,37 @@ function start () {
     }
 
     argv.inputPath = path.isAbsolute(argv.input) ? argv.input : path.join(process.cwd(), argv.input)
-    const results = require(argv.inputPath)
-    const report = buildReport(results)
-    writeReport(report, argv.outputPath, (err) => {
-      if (err) console.err('Error writting report: ', err)
-      else console.log('Report written to: ', argv.outputPath)
-    })
-  } else {
-    const concatStream = concat((res) => {
-      const results = JSON.parse(res.toString())
-      const report = buildReport(results)
-      writeReport(report, argv.outputPath, (err) => {
-        if (err) console.err('Error writting report: ', err)
-        else console.log('Report written to: ', argv.outputPath)
+    if (argv.compare) {
+      steed.map(argv.compare, (val, cb) => {
+        val = path.isAbsolute(val) ? val : path.join(process.cwd(), val)
+        fs.access(val, fs.F_OK, function (err) {
+          if (err) return cb(new Error('Can\'t access ' + val))
+          cb(null, require(val))
+        })
+      }, (err, compare) => {
+        if (err) return console.log(err)
+        compare = sort(compare)
+        const results = require(argv.inputPath)
+        const report = buildReport(results, compare)
+        writeReport(report, argv.outputPath, (err) => {
+          if (err) console.err('Error writting report: ', err)
+          else console.log('Report written to: ', argv.outputPath)
+        })
       })
-    })
-    process.stdin.pipe(concatStream)
+    }
+  } else {
+    let compare = []
+    process.stdin
+     .pipe(ndjson.parse())
+     .on('data', (json) => { compare.push(json) })
+     .on('finish', () => {
+       compare = sort(compare)
+       const report = buildReport(compare[compare.length - 1], compare)
```

Comment:
Does this mean that the most recent run is the last index? could you change this to sort with the most recent to be first?